### ISSUE TITLE
Implementation in tf.keras

### ISSUE THREAD

**Issue by Jorisfournel** [maunzzz/caffe-crfgd#1]
Hi,

I would like to congratulate you for your work,
I work with medical images and I'd like to see if such
a post processing unit would be useful for segmentation 
task in the field, could you point to me the essential elements
of the coding under Keras for example ? I will be trying to 
figure it out from your paper, but it seems that there is 
structural plus gradient computation and updating specificities to implement.

Anyway, congrats and thanks for the contribution,
Joris Fournel

**Comment by maunzzz** [comment]
Hi Joris!

Thank you for your interest in the work.
I have not gotten around to implement it in any other framework but I believe it should be kind of straight-forward given that keras and pytorch have automatic differentiaion. For implementation details I would suggest having a look at the code in this repository and the paper (also the supplementary material).

Good luck!
